mod2/src/main/java/org/example/ParserJson.java

```diff
@@ -0,0 +1,49 @@
+package org.example;
+
+import com.google.gson.*;
+
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+/*
+Application should read a config file on the startup
+Then it should ensure that all of files from the config exist
+Then it should rename each file adding a suffix from the config to its name
+It should print the results of the renaming like:
+old_name -> new_name
+ */
+
+public class ParserJson {
+
+    public Configuration fromJSON() {
+        Configuration fileName = null;
```

Comment:
Not a good parameter name. It's mostly about configuration not about filename(one filename???)